proxy/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM nginx
+
+COPY nginx.conf /etc/nginx/nginx.conf
```

Comment:
has this been tested? the nginx.conf is located in the proxy folder, i would imagine this fails.